### PR TITLE
feat(schematics): update @angular/cli to 6.2.3 and ngrx 6.1.0

### PR DESCRIPTION
**Pull request by FrozenPandaz**
## Current Behavior

Nx is incompatible with `@angular/cli@^6.2.0`.

Nx is using `@ngrx/*@6.0.1`

## Expected Behavior

Nx is compatible, generates projects, and uses `@angular/cli@6.2.3`.

Nx uses `@ngrx/*@6.1.0`

Other notable package updates to stay consistent with the versions generated by `@angular/cli@6.2.3`:
* `karma@~3.0.0`
  * Fixes security vulnerability :)
* `protractor@~5.4.0`
* `ts-node@~7.0.0`
* `tslint@~5.11.0`

### CODE REVIEW

File: e2e/schematics/ngrx.test.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ describe('ngrx', () => {
       runCLI(
         'generate ngrx users --module=apps/myapp/src/app/app.module.ts --root --collection=@nrwl/schematics'
       );
+      copyMissingPackages();
```
Comment (vsavkin) [inline e2e/schematics/ngrx.test.ts]: why is this needed?

File: e2e/utils.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ export function copyMissingPackages(): void {
     'jasmine-marbles',
     '@nrwl',
     'angular',
+    '@angular/cli',
```
Comment (vsavkin) [inline e2e/utils.ts]: why is this needed?

**Review by vsavkin — APPROVED** [review]

File: e2e/schematics/ngrx.test.ts (hunk 1 above)
Comment (FrozenPandaz) [reply e2e/schematics/ngrx.test.ts]: This is needed because generating ngrx installs packages and therefore wipes the @nrwl dependencies. This copies it back
